source/reactions.yaml

```diff
@@ -0,0 +1,111 @@
+  '/teams/{team_id}/channels/{channel_id}/posts/{post_id}/save_reaction':

```

Comment:
I'm going to make this route `/reactions/save` just because duplicate calls are considered as updates  (even though there's nothing to update with reactions) instead of errors